torch/_inductor/ir.py

```diff
@@ -5817,6 +5817,7 @@ def codegen(self, wrapper) -> None:  # type: ignore[no-untyped-def]
             if kernel.arg_names.index(kwarg) in kernel.constexprs:
                 constexpr_indices.append(idx)
 
+        equal_to_1 = triton_meta["configs"][0].equal_to_1
```

Comment:
note: we need to remove this. In future triton versions (i.e. branch cut  - end of next week) triton_meta["configs"][0].equal_to_1 will not exist)